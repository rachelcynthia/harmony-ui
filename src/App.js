import React from 'react';
import './App.css';
import Header from './components/Header/Header';
import Login from './components/Login/Login';

function App() {
  return (
    <>
      <Header isLoggedIn={false}/>
      <Login isLoggedIn={false} />
    </>
  );
}

export default App;
