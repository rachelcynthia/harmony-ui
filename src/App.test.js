import React from 'react';
import TestRenderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const app = TestRenderer.create(<App />).toJSON();
  expect(app).toMatchSnapshot();
});
