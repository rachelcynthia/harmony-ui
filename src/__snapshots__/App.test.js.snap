// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Array [
  <nav>
    <div
      className="left-links"
    >
      <img
        alt="Logo"
        height="50"
        src="test-file-stub"
        width="50"
      />
    </div>
    <div
      className="centered-links"
    >
      <a
        className="links"
      >
        Harmony
      </a>
    </div>
    <div
      className="right-links"
      style={
        Object {
          "width": "50px",
        }
      }
    />
  </nav>,
  <div
    className="login_container"
  >
    <div
      className="login_component"
    >
      <div
        className="login_header"
      >
        Please login to continue
      </div>
      <div
        className="login_body"
      >
        <input
          className="input_text"
          placeholder="Enter email"
          type="text"
        />
        <input
          className="input_text"
          placeholder="Enter password"
          type="password"
        />
        <input
          className="input_button"
          onClick={true}
          type="button"
          value="LOGIN"
        />
      </div>
      <div
        className="register_component"
      >
        New to Harmony? 
        <a
          className="register_button"
          href="#"
        >
          Register
        </a>
      </div>
    </div>
  </div>,
]
`;
